packages/preload/src/modules/workspace.ts

```diff
@@ -239,26 +248,44 @@ export async function getAvailable(_name: string = NEW_SCENE_NAME) {
  * @param {string} [opts.repo] - The repository to use as the template. Defaults to an empty scene template repository.
  * @returns {Promise<Project>} - A promise that resolves to the created project.
  */
-export async function createProject(opts?: { name?: string; repo?: string }): Promise<Project> {
-  const { name, repo } = {
-    name: opts?.name ?? NEW_SCENE_NAME,
-    repo: opts?.repo ?? EMPTY_SCENE_TEMPLATE_REPO,
-  };
+export async function createProject(opts?: {
+  name?: string;
+  path?: string;
+  repo?: string;
+}): Promise<Project> {
+  const { name, path: projectPath } =
+    opts?.path && opts?.name
+      ? { name: opts.name, path: opts.path }
+      : await getAvailable(opts?.name ?? NEW_SCENE_NAME);
+
+  const templateRepo = opts?.repo ?? EMPTY_SCENE_TEMPLATE_REPO;
+
+  try {
+    await fs.mkdir(projectPath);
```

Comment:
this is an edge case, but if you use in your path a grandchild folder, the scene creation flow breaks

I think you can use `fs.mkdir(projectPath, { recursive: true })`

https://github.com/user-attachments/assets/0326a732-b8f5-4c6e-ba5a-6df567078625

Reply:
yes! fixed